packages/lib/src/interactions/AxialSelectToZoom.tsx

```diff
@@ -44,7 +46,7 @@ function AxialSelectToZoom(props: Props) {
       axis={axis}
       id={`${axis.toUpperCase()}SelectToZoom`}
       modifierKey={modifierKey}
-      disabled={disabled}
+      disabled={visRatio !== undefined || disabled}
```

Comment:
Okay, so this is the first case of a component deciding, by itself, how to behave based on `visRatio`. Here, it's basically disabling itself inside a canvas with an `'equal'` or custom `aspect`.

Note that, thanks to `|| disabled`, the consumer still has the ability to disable the interaction when used inside a canvas with `aspect="auto"`; but the opposite is not true: the consumer cannot force-enable the interaction inside a canvas with an `'equal'` or custom `aspect`, since this use case is not clearly defined/supported.